### ISSUE TITLE
Change Days

### ISSUE THREAD

**Issue by ningweiii** [Yehzuna/schedule#16]
hi Yehzuna, i wonder is it possible to modify days of the week from 7days to 5 days? by default, you set 7 days.

**Comment by Yehzuna** [comment]
Added to my todo list :)

**Comment by ningweiii** [comment]
I am trying to use your scheduler into asp .net, but there is always an error called "$(...).jqs is not a function"
I did include what you mentioned in the readme file, 
```
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>    
    <script src="dist/jquery.schedule.min.js"></script>
    <link rel="stylesheet" href="dist/jquery.schedule.min.css">
```

what did I miss?

**Comment by evanradcliffe** [comment]
@NingPekin I had a similar problem though I'm using Spring Boot. I had to add the path to where I placed the library files to the resource locations. [This page](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-2.1&tabs=aspnetcore2x) might help you. It's referred to as serving static content.

**Comment by Yehzuna** [comment]
Added today with the v2.2.0 !
